src/pages/WebMeetWaitingRoom.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from "react";
+import "../App.css";
```

Comment:
does this need to imported here? don't we already import this in `App.js`?

Reply:
import removed